### PR TITLE
[MKC-1763] - Update "Arduino Web editor"

### PR DESCRIPTION
**Pull request by pedromsousalima**
## What This PR Changes
- Update "Arduino Web editor" images were updated to the latest editor UI and the platform name was refreshed

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/learn/01.starting-guide/03.the-arduino-web-editor/assets/finding-an-example.png
Comment (jacobhylen) [inline content/learn/01.starting-guide/03.the-arduino-web-editor/assets/finding-an-example.png:1]: can we make these 1080p?

File: content/learn/01.starting-guide/03.the-arduino-web-editor/assets/web-editor.png
Comment (jacobhylen) [inline content/learn/01.starting-guide/03.the-arduino-web-editor/assets/web-editor.png:1]: this too

File: content/learn/01.starting-guide/03.the-arduino-web-editor/arduino-web-editor.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+description: 'The Arduino Cloud Editor allows you to write code and upload sketches to any official Arduino board from your web browser.'
 author: 'Liam Aljundi'
-tags: [Web Editor, Online Coding]
+tags: [Cloud Editor, Online Coding]
 ---
 
-The Arduino Web Editor allows you to write code and upload sketches to any official Arduino board directly from your web browser (Chrome, Firefox, Safari and Edge). However, we recommend you use Google Chrome. 
+The Arduino Cloud Editor allows you to write code and upload sketches to any official Arduino board directly from your web browser (Chrome, Firefox, Safari and Edge). However, we recommend you use Google Chrome. 
 
 This IDE (Integrated Development Environment) is part of [Arduino Create](https://create.arduino.cc/), an online platform that enables developers to write code, access tutorials, configure boards, and share projects. Designed to provide users with a continuous workflow, Arduino Create connects the dots between each part of a developer's journey from inspiration to implementation. Meaning, you now have the ability to manage every aspect of your project right from a single dashboard.
```
Comment (jacobhylen) [inline content/learn/01.starting-guide/03.the-arduino-web-editor/arduino-web-editor.md:10]:
```suggestion
This IDE (Integrated Development Environment) is part of [Arduino Cloud](https://app.arduino.cc/), an online platform that enables developers to write code, access tutorials, configure boards, and share projects. Designed to provide users with a continuous workflow, Arduino Cloud connects the dots between each part of a developer's journey from inspiration to implementation. Meaning, you now have the ability to manage every aspect of your project right from a single dashboard.
```

**Review by jacobhylen — COMMENTED** [review]
Thank you @pedromsousalima 
I've left a couple of comments, when these are fixed we should be able to merge.

File: content/learn/01.starting-guide/03.the-arduino-web-editor/assets/finding-an-example.png
Comment (pedromsousalima) [reply content/learn/01.starting-guide/03.the-arduino-web-editor/assets/finding-an-example.png:1]: Done

File: content/learn/01.starting-guide/03.the-arduino-web-editor/assets/web-editor.png
Comment (pedromsousalima) [reply content/learn/01.starting-guide/03.the-arduino-web-editor/assets/web-editor.png:1]: Done

**Review by jacobhylen — APPROVED** [review]
lgtm

**Pull request merged**